index.html

```diff
@@ -1451,7 +1451,15 @@ <h2>Core Vocabulary Definitions</h2>
           <code>unsubscribeevent</code>, or both terms within an
           <a href="#dfn-array" class="internalDFN" data-link-type="dfn">Array</a>.</span></p><p class="note">
 		  It is considered to be good practice that each <code>subscribeevent</code> has a corresponding <code>unsubscribeevent</code> unless the protocol
-		  supports implicit unsubscription mechanisms (e.g., heartbeat to detect connection loss).</p></section>
+		  supports implicit unsubscription mechanisms (e.g., heartbeat to detect connection loss).</p>
+          <p>
+            EventAffordances are similar to observable PropertyAffordances in the sense that the Thing itself informs interested Consumers about state changes.
+            However, a main difference of EventAffordances is that not every change of the associated resource needs to trigger an event message to be emitted, e.g. a critical threshold for a numeric value.
+        Furthermore, EventAffordances allow for more complex subscribing and unsubscribing mechanisms via the definition of <code>subscription</code>, <code>dataResponse</code>, and <code>cancellation</code> DataSchema definitions.
+            Not all protocols might support these more advanced mechanisms, however, which is why in some scenarios, events may be very similar to observable PropertyAffordances.
+            In these cases, the choice between a Property and a Event for modelling the Affordance should be made based on the semantics of the underlying resource.
```

Comment:
Thank you for your feedback, @egekorkan! I tried to address [your comment](https://github.com/w3c/wot-thing-description/pull/2018/files#r1602172783) with the following:

```suggestion
            In these cases, the choice between a Property and a Event for modelling the Affordance should be made based on the semantics of the underlying resource; for example, if the state of the affordance is also supposed to be read or written by a Consumer, then a Property is most likely the appropriate choice.
```

Reply:
Sorry for suggesting in the wrong document as well :) I like the wording and committed the suggestion

Reply:
Thank you! :) I've now also added the lines to the source file, which actually did not change the rendered version :)